arangod/Agency/Constituent.cpp

```diff
@@ -548,12 +545,7 @@ void Constituent::callElection() {
     }
     // Count the vote as a nay
     if (++nay >= majority) {                  // Network: majority against?
```

Comment:
"++nay >= majority" is wrong.  Should be "++nay>(size() - majority)".  Why?  if size() is 4, majority is 3.  2 nay votes are all that are needed to invalidate the election.

Reply:
Good catch. The current implementation only allows for odd agency sizes, but we are going to change this. So it is good to fix it right now and here. Done.